modules/software-extensibility/pages/bonita-repository-access.adoc

```diff
@@ -0,0 +1,149 @@
+= Access to Bonita Artifact Repository (Subscription editions only)
+:page-aliases: ROOT:bonita-repository-access.adoc
+:description: Set up access to Bonita Artifact Repository to download development dependencies.
+
+{description}
+
+[#credentials]
+== Ask for credentials to your account manager
+
+Once your subscription is registered, your account manager will automatically receive an email with valid credentials.
+These credentials will last as long as your subscription, but you will not need to change them if you extend that subscription.
+
+Credentials are unique per subscription and include:
+
+* A login (the email used for the subscription)
+* A token
+
+== Maven
+
+Starting from Bonita 2021.2, Bonita Studio uses https://maven.apache.org/[maven] internally to resolve required dependencies.
+
+[NOTE]
+====
+In the below example, we use Maven as the project build tool for extension projects. But it is also possible to use Gradle if it is the tool of your choice.
+====
+
+An Internet connection is required.
+
+=== Maven settings overview
+
+Once you were granted with Bonita Artifact Repository credentials, you have to tell https://maven.apache.org/configure.html[configure] `maven` to use Bonita release repository.
+
+This is done in a file named https://maven.apache.org/settings.html[settings.xml] which is present by convention in the following default place: `$HOME/.m2/settings.xml`
+
+[source, xml]
+.Sample sections of settings.xml
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 https://maven.apache.org/xsd/settings-1.0.0.xsd">
+  <localRepository/>
+  <interactiveMode/>
+  <offline/>
+  <pluginGroups/>
+  <servers/>
+  <mirrors/>
+  <proxies/>
+  <profiles/>
+  <activeProfiles/>
+</settings>
+----
+
+=== Bonita repositories declaration
```

Comment:
This section lack terminology homogeneity. We can find the different terms "Bonita repositories", "Bonita release repository", and "Bonitasoft releases"